MontyTRC89/TombEngine
# Savegame functions

This pr aims to implement 3 functions: Save, Load and Delete:

Save - saves the savegame provided the slot

Load - loads the savegame provided the slot

Delete - Deletes the savegame file provided the slot

things to test:
- check that all 3 funcs work correctly
- Check that OnSave and OnLoad work correctly after calling Save and Load in the script
- check that the else part is triggered in Delete function
- check that the savegame is deleted from Saves folder correctly
- (optionally since I encountered) check that the game doesn't crash when entering the load page in title (this will probably happen after misuse of function.)

(thanks to Lwmte who helped with Load function :) )
